extensions/git/src/commands.ts

```diff
@@ -608,6 +608,9 @@ export class CommandCenter {
 
 				uris = resources.map(r => r.resourceUri);
 			}
+			else {
+				uris = [(window.activeTextEditor && window.activeTextEditor.document.uri)] as Uri[];
```

Comment:
If window.activeEditor is undefined, this will cause `uris` to be an array with one element (undefined). Better use the ternary operator.

```
uris = (activeTextEditor ? [activeTextEditor.document.uri] : []) as Uri[];
```

Also you can move the `activeTextEditor` declaration above the uris declaration (close to the start of the `openFile` function), and reuse that.

Reply:
Sorry, for the late response. You're right about the check. But I doubt that this command will be invoked without an active editor. Because this command would not be accessible when there is no active editor. Let me know if I'm wrong.

Reply:
Actually the `window.activeTextEditor` is a nullable field (of type `TextEditor | undefined`) hence using the existence check is required, otherwise there can be bugs in the future.